src/Entities/Player.h

```diff
@@ -138,6 +138,15 @@ class cPlayer :
 
 	// tolua_begin
 
+	/** Prevent the palyer from moving and lock him into a_Location. */
+	void Freeze(const Vector3d & a_Location);
```

Comment:
This should be enough. However, I question the parameter, why is a position needed? Shouldn't the current position be the one?